keps/7610-default-local-queue-creation/kep.yaml

```
@@ -0,0 +1,37 @@
+title: Create Default LocalQueues Based on NamespaceSelector
+kep-number: 7610
+authors:
+  - "@j-skiba"
+status: provisional
+creation-date: 2025-11-12
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.2"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v0.15"
+  beta: "v0.15"
+  stable: "v0.15"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DefaultLocalQueue
```

```suggestion
  - name: DefaultLocalQueueCreation
```
Maybe to make it less confusing with the other pre-existing KEP LocalQueueDefaulting